docs/methods/sampling/ehrenfest.md

```diff
@@ -1,3 +1,523 @@
+equations are valid for the exact wavefunctions. In the Kohn-Sham approach, the wavefunctions are
+replaced by a linear combination of basis functions. For plane waves the equations remain the same,
+since they do not depend on the nuclear coordinates and therefore are independent of the change of
+the ionic positions during MD. The case of a Gaussian basis set requires a more detailed analysis.
+Using Gaussian basis sets for the expansion of the wavefunctions, an implicit dependence of the
+wavefunctions on the nuclear positions is introduced. Since in an MD approach the nuclear positions
+are function of time, the time derivative in the quantum mechanical action has to be replaced by the
+total time derivative
+
+$$
+\frac{d}{dt} = \frac{\partial}{\partial t} + \sum_A \frac{\partial {\bf R}_A}{\partial t}\frac{\partial}{\partial {\bf R}_A}.
+$$
+
+Due to the introduction of the basis, the independent variables for making the action constant
+become the expansion coefficients $C_{j\alpha}(t)$ of the molecular orbital $j$ in the contracted
+basis functions $\phi_\alpha({\bf r},t)$
+
+$$
+\dot{C}_{j\alpha} = -\sum_{\alpha\beta}S^{-1}_{\beta\gamma}\left(  i H_{\beta\gamma} + B_{\beta\gamma} \right) C_{j\gamma}
+$$
+
+where
+
+$$
+S_{\alpha\beta} = \langle \phi_\alpha| \phi_\beta \rangle \qquad B_{\alpha\beta} = \langle \phi_\alpha | \frac{d}{dt} \phi_{\beta} \rangle
+$$
+
+Hence, in Ehrenfest dynamics an additional contribution due to the derivative of the basis functions
+becomes part of the Hamiltonian used in the exponential operator. Instead of being purely imaginary,
+the matrix in the exponential of the propagator becomes fully complex.
+
+## Running real time time dependend DFT in CP2K
+
+To run RTP or Ehrenfest MD with CP2K, the corresponding [RUN_TYPE](#CP2K_INPUT.GLOBAL.RUN_TYPE) in
+the \[GLOBAL\] (#CP2K_INPUT.GLOBAL) section has to be selected, and the [MD](#CP2K_INPUT.MOTION.MD)
+section has to be present to specify the time step length `TIMESTEP` and the desired number of steps
+(`STEPS`). It is crucial to set an appropriate time step to propagate the electronic degrees of
+freedom, i.e., in the order of atto-seconds. All other input parameters related to the RT-TDDFT run
+are specified from the section
+[REAL_TIME_PROPAGATION](#CP2K_INPUT.FORCE_EVAL.DFT.REAL_TIME_PROPAGATION). The simulation needs to
+start from the electronic density at $t_0$ (`INITIAL_WFN`). This can be the ground state obtained by
+means of an initial SCF optimization (`SCF_WFN`) or providing the restart file of a previously
+computed state (`RESTART_WFN`). The propagation can also be restarted setting the initial
+wavefunction as `RT_RESTART` and providing the correct restart file, which has a different format
+with respect to the SCF restart file. The path to the restart file is in both cases provided by the
+usual [RESTART_FILE_NAME](#CP2K_INPUT.FORCE_EVAL.DFT.RESTART_FILE_NAME) key.
```

Comment:
Aliases don't work, you have to refer to keywords by their primary name.
In this case it's `#CP2K_INPUT.FORCE_EVAL.DFT.WFN_RESTART_FILE_NAME`

Reply:
Is it not possible just to have it written with different font, without linking the keyword in the input?

Reply:
You could format it as code by putting it in blockquotes:

```
`RESTART_FILE_NAME`
```

Or you could keep it as a link like this:
```
[RESTART_FILE_NAME](#CP2K_INPUT.FORCE_EVAL.DFT.WFN_RESTART_FILE_NAME)
```

Reply:
I think I did exactly that,  but there was a part of the name missing